spec/rspec/core/formatters/html_formatter_spec.rb

```diff
@@ -41,6 +25,12 @@ module Formatters
           allow(RSpec.configuration).to receive(:load_spec_files) do
             RSpec.configuration.files_to_run.map {|f| load File.expand_path(f) }
           end
+
+          # This is to minimize churn on backtrace lines
+          allow_any_instance_of(Notifications::FailedExampleNotification).
```

Comment:
Given we recommend against `allow_any_instance_of, it's worth considering alternatives here.  Can the interface be improved to make this simpler?  Or can we use `stub_const` to replace a constant with an alternate implementation for this test?